src/data/index.js

```diff
@@ -36,6 +36,9 @@ const books = [
     title: 'ES6 & Beyond',
     chapters: ES6BeyondChapters,
   },
-];
+].map((book, i) => {
```

Comment:
I don't think this change is necessary. Unless there was some sort of warning you were trying to fix?